modules/brainbrowser/js/brainbrowser.loris.js

```diff
@@ -748,17 +749,52 @@ $(function() {
         minc_ids_arr = [minc_ids];
     }
 
+    var request;
```

Comment:
should probably use `let` instead of `var` for any new code in general.

Reply:
Oups. I read about that after having sent the PR ;). I changed the occurrences in the code I inserted but `var` is used all over BrainBrowser's code. Should probably be taken care of in another PR? What do you think @driusan?